src/Akka.Persistence.Sql/Config/JournalTableConfig.cs

```diff
@@ -40,7 +40,7 @@ public JournalTableConfig(Configuration.Config config)
 
         public string SchemaName { get; }
 
-        // TODO: implement this settings
+        [Obsolete("Not implemented")]
         public bool UseEventManifestColumn { get; } = false;
```

Comment:
Why is this obsolete?

Reply:
It was never implemented, configuration wise.

Reply:
It was me using Obsolete as a debugging tool, i'll turn it back on